modules/RTCStats/interfaces.ts

```diff
@@ -7,5 +7,13 @@ export interface IRTCStatsConfiguration {
         rtcstatsSendSdp?: boolean;
         rtcstatsStoreLogs?: boolean;
         rtcstatsUseLegacy?: boolean;
-    };
+    },
+    confName: string;
 }
+
+export interface ITraceOptions {
+    endpoint: string;
+    isBreakoutRoom: boolean;
+    meetingFqn: string;
+    useLegacy: boolean;
```

Comment:
How long do we need to support this legacy thing?

Reply:
We don't need it anymore, I'll check if it has any impact with the rtcstats module so that I can remove it.